specs/category.spec.ts

```diff
@@ -0,0 +1,63 @@
+describe('InstantSearch - Search on specific category', () => {
+  it('navigates to the e-commerce demo', async () => {
+    await browser.url('examples/e-commerce/');
+  });
+
+  it('selects "Appliances" category in list', async () => {
+    await browser.setSelectedHierarchicalMenuItem('Appliances');
+  });
+
+  it('selects "Small Kitchen Appliances" category in list', async () => {
+    await browser.setSelectedHierarchicalMenuItem('Small Kitchen Appliances');
+  });
+
+  it('must have the expected results for "Small Kitchen Appliances"', async () => {
+    const hitsTitles = await browser.getHitsTitles();
+
+    expect(hitsTitles).toEqual([
+      'Insignia™ - 2.6 Cu. Ft. Compact Refrigerator - Black',
+      'Keurig - K50 Coffeemaker - Black',
+      'Keurig - K200 Brewer - Black',
+      'Frigidaire - 3.3 Cu. Ft. Compact Refrigerator - Stainless Steel',
+      'Ninja - Mega Kitchen System 72-Oz. Blender - Black',
+      'Ninja - Nutri Ninja Pro 24-Oz. Blender - Black/Silver',
+      'Oster - Inspire 2-Slice Wide-Slot Toaster - Black',
+      'Keurig - The Original Donut Shop K-Cup® Pods (18-Pack)',
+      'Chefman - Express Air Fryer - Black',
+      'Oster - 10-Speed Blender - Black',
+      'Keurig - Swiss Miss Milk Chocolate Hot Cocoa K-Cup® Pods (16-Pack)',
+      'Keurig - Cinnabon Classic Cinnamon Roll K-Cup® Pods (18-Pack) - Multi',
+      'Ninja - Nutri Ninja 32 Oz. Auto-iQ Blender - Black',
+      'Anova - Precision Cooker WiFi',
+      'Nespresso - VertuoLine Evoluo Espresso Maker/Coffeemaker - Gray',
+      'Magic Bullet - Blender - Silver/Black',
+    ]);
+  });
+
+  it('deselects "Small Kitchen Appliances" category in list', async () => {
+    await browser.setSelectedHierarchicalMenuItem('Small Kitchen Appliances');
```

Comment:
the `setXX` functions should probably be called "click", "toggle" or "refine"

Reply:
Addressed in https://github.com/algolia/instantsearch-e2e-tests/pull/5